starter-code/app.js

```diff
@@ -45,5 +49,7 @@ app.use(function(err, req, res, next) {
   res.status(err.status || 500);
   res.render("error");
 });
-
+app.listen(3000, () => {
```

Comment:
You don't need this, since the express-generator project already does this connection for you, check out the `www` file. In fact, this is making the app crash, since it's trying to connect to a port already in use.